data.sql

```diff
@@ -41,3 +41,31 @@ UPDATE animals SET owner_id =(SELECT id FROM owners WHERE full_name = 'Bob') WHE
 UPDATE animals SET owner_id =(SELECT id FROM owners WHERE full_name = 'Melody Pond') WHERE name = 'Squirtle' OR name = 'Charmander' OR name = 'Blossom';
 UPDATE animals SET owner_id =(SELECT id FROM owners WHERE full_name = 'Dean Winchester') WHERE name = 'Angemon' OR name = 'Boarmon';
 
```

Comment:
- [ ] It would be better if our README file includes instructions about how to clone and run the app locally. By fixing this our repo looks professional :smile: